app/main.dev.ts

```diff
@@ -117,3 +124,27 @@ app.on('activate', () => {
 app.on('open-file', (_, path) => {
   if (mainWindow !== null) mainWindow.webContents.send('open-file', path);
 });
+
+autoUpdater.on('update-available', () => {
+  if (mainWindow !== null)
+    mainWindow.webContents.send(
+      'message',
+      'Update available. Keep your app opened. You will be informed when it is downloaded.'
```

Comment:
when it **will** be downloaded ?
```suggestion
      'Update available. Keep your app opened,  you will be informed when it will be downloaded.'
```